content/run.md

```diff
@@ -143,7 +143,7 @@ There are dedicated tools and workflow for running single-cell simulations over
 Follow the below procedure to run CTSM over a specific region of interest for a specific resolution. 
 
 ### Domain and surface data
-To run over the Scandinavia region (latitude 48N to 81N, longitude 4E to 42E) at 0.5 degree resolution you first need to produce the domain and surface data files for the region using the python script `subset_data.py` available under the CTSM tools directory `CTSM_ROOT/tools/contrib/subset_data.py`. The python script file can be found [here](https://github.com/ESCOMP/CTSM/blob/master/tools/site_and_regional/subset_data.py). 
+To run over the Scandinavia region (latitude 48N to 81N, longitude 4E to 42E) at 0.5 degree resolution you first need to produce the domain and surface data files for the region using the python script `subset_data.py` available under the CTSM tools directory `CTSM_ROOT/tools/site_and_regional/subset_data.py`. The python script file can be found [here](https://github.com/ESCOMP/CTSM/blob/master/tools/site_and_regional/subset_data.py). 
```

Comment:
Now we have a duplicate - the last two sentences point to the same. I would suggest to merge them and either mention the subdirectory or point to the github page. I tend to the second option, as it is easier to check the validity of a link.

Reply:
I think its fine to have that additional information. Link provides to have a quick glance of the script. if its just a link then users may think of downloading it.

Reply:
OK. But it is still more work to keep it up-to-date :-).